addons/mrp/models/stock_rule.py

```diff
@@ -64,9 +64,10 @@ def _run_manufacture(self, procurements):
                 origin_production = production.move_dest_ids and production.move_dest_ids[0].raw_material_production_id or False
                 orderpoint = production.orderpoint_id
                 if orderpoint:
-                    production.message_post_with_view('mail.message_origin_link',
-                                                      values={'self': production, 'origin': orderpoint},
-                                                      subtype_id=self.env.ref('mail.mt_note').id)
+                    production.message_post(
```

Comment:
Keep the link if the orderpoint is not generated from replenish report

Reply:
yes, message_post_with_view is already there, which will keep the link if origin_production will be there.